curbmap-ios-mvp/Model/User.swift

```diff
@@ -4,24 +4,24 @@
 //
 //  Copyright © 2017 curbmap. All rights reserved.
 //
-
 import Foundation
+import KeychainAccess
 import MapKit
 
-let currentUser = User(username: "curbmap", password: "TestCurbm@p1");
-
-class User {
+class User : NSObject {
+    public static let currentUser = User(username: "curbmap", password: "TestCurbm@p1");
```

Comment:
This is fine for now, but eventually we'll need to refactor so that currentUser only instantiate after a user signs up or log in

Reply:
No, we need that curbmaptest user to be always logged in if a user has not been created. Otherwise, we'll never have a JWT for auth.